ui/app/components/pki/config-pki.js

```diff
@@ -39,6 +40,10 @@ export default Component.extend({
   actions: {
     save(section) {
       this.set('loading', true);
+      if (section === 'crl') {
+        // set 'disable' here from buildCrl toggle state
+        this.config.set('disable', !this.buildCrl);
```

Comment:
This feels like form data cleanup that should instead live in the serializer? Particularly because we're changing a value that the set value is based on, so it feels very circular. It's also very hard to reason about (which I'm sure is true to reality since it's PKI)